### ISSUE TITLE
A11y_CDE - Mission Control and APIs_Measure Failure Investigation-Measure Drilldown_Screen Reader: Narrator is just reading as Group for Add dimension drop down.

### ISSUE THREAD

**Issue by Bhaskar301101** [facebook/react#22225]
Check out Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.

Environment Details:
OS: Windows 20H2 1903(OS Build 19042.508)​
Microsoft Edge Dev Version 93.0.926.0 (Official build) dev (64-bit)
URL:  https://mission-control-ci.azurewebsites.net/​
​
Repro Steps:​​
1) Hit the URL : https://mission-control-ci.azurewebsites.net/​
2) Login with Valid login credentials​​
3) Start Narrator
4) Home page will get open ​​
5) Tab till left navigation bar and Press enter on Windows icon​​
6) Tab till Release Readiness (RQV) and press enter ​​
7) Tab till Compliance from left Navigation bar  and press enter ​​
8) Tab till Compliance expandable button present in the page and press enter​​
9) Tab till any graph and press enter​
10) Graph pop-up will get open​
11) Tab till Investigate Measure Link and press enter
12) Measure Failure Investigation page will get open 
13) Tab till Jump to related views (Three dots) and press enter
14) Tab till Data Landscape and press enter
15) Measure: 20203773 - [RQV Compliance Measure] Privacy - Reset page will get open
16) Tab till Add dimension drop down and check whether Narrator is reading as Group or not

Actual Result:​​
Narrator is not reading descriptive information when focus land on the 'Select a dimension' combo box. It is announcing add dimension edit scan off space space 0 result available select is focus to find the dimension list......... 

Expected Result:​​
Narrator should read descriptive information when focus land on the 'Select a dimension' combo box. It should announce 'Select a dimension combo box collapsed editable' 

Note: Same issue repro with NVDA & JAWS screen reader as well.
NVDA is narrating 'Add dimension edit has auto complete' and JWAS is narrating 'Add dimension column edit type and text'.

User Impact:​​
Screen Reader users will have difficulty in understanding if Narrator is just reading as Group for Add dimension drop down with out reading the role assigned to the drop down

MAS Reference:​​
(https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}[](url)
<img width="942" alt="BUG1" src="https://user-images.githubusercontent.com/89897257/131650206-60865ec3-39e4-4d1a-948b-197dc9a6ca94.png">

**Comment by eps1lon** [comment]
This repository is not affiliated with https://mission-control-ci.azurewebsites.net/%E2%80%8B
